package.json

```diff
@@ -2790,40 +2790,103 @@
-					"gitlens.focus.allowMultiple": {
+					"gitlens.launchpad.allowMultiple": {
 						"type": "boolean",
 						"default": true,
 						"markdownDescription": "Specifies whether to allow opening multiple instances of the _Launchpad_ as an editor tab",
 						"scope": "window",
 						"order": 10
 					},
-					"gitlens.focus.experimental.indicators.enabled": {
+					"gitlens.launchpad.ignoredRepositories": {
+						"type": "array",
+						"default": [],
+						"items": {
+							"type": "string"
+						},
+						"markdownDescription": "Specifies the repositories to ignore in the _Launchpad_",
+						"scope": "window",
+						"order": 19
+					},
+					"gitlens.launchpad.staleThreshold": {
+						"type": [
+							"number",
+							"null"
+						],
+						"default": null,
+						"markdownDescription": "Specifies the number of days after which a pull request is considered stale and moved to Other in the _Launchpad_",
+						"scope": "window",
+						"order": 20
+					},
+					"gitlens.launchpad.indicator.enabled": {
 						"type": "boolean",
 						"default": false,
-						"markdownDescription": "(Experimental) Specifies whether to enable status bar indicators for _Launchpad_",
+						"markdownDescription": "Specifies whether to enable status bar indicator for _Launchpad_",
 						"scope": "window",
 						"order": 21
 					},
-					"gitlens.focus.experimental.indicators.openQuickFocus": {
+					"gitlens.launchpad.indicator.openInEditor": {
 						"type": "boolean",
-						"default": true,
-						"markdownDescription": "(Experimental) Specifies whether to open _Launchpad_ as an editor tab or quickpick command when clicking on the indicators",
+						"default": false,
+						"markdownDescription": "Specifies whether to open _Launchpad_ as an editor tab when clicking on the status bar indicator",
 						"scope": "window",
 						"order": 22
 					},
-					"gitlens.focus.experimental.indicators.data.enabled": {
+					"gitlens.launchpad.indicator.polling.enabled": {
 						"type": "boolean",
 						"default": true,
-						"markdownDescription": "(Experimental) Specifies whether the status bar indicators will fetch data for _Launchpad_.",
+						"markdownDescription": "Specifies whether the status bar indicator will fetch and display pull request data for _Launchpad_.",
 						"scope": "window",
 						"order": 23
 					},
-					"gitlens.focus.experimental.indicators.data.refreshRate": {
+					"gitlens.launchpad.indicator.polling.interval": {
 						"type": "number",
 						"default": 30,
-						"markdownDescription": "(Experimental) Specifies the rate (in minutes) at which to refresh the status bar indicators for _Launchpad_. Use 0 to disable automatic refreshing",
+						"markdownDescription": "Specifies the rate (in minutes) at which the status bar indicator will fetch pull request data for _Launchpad_. Use 0 to disable automatic polling",
 						"scope": "window",
 						"order": 24
+					},
+					"gitlens.launchpad.indicator.showTopItem": {
```

Comment:
What do you think of something like `gitlens.launchpad.indicator.style`

`icon` -- rocket icon
`priorityItem` (don't love the name at all, but can't think of something better off-hand) -- rocket icon + priority item name
`priorityGroupIcon` -- rocket icon if no priority group, otherwise the group icon?

Thoughts?

Reply:
So `priorityItem` would be the equivalent of `showTopItem: true`, `icon` would be the equivalent of `showTopItem: false`, and `priorityGroupIcon` is similar to `showTopItem: false` except the icon itself changes with the top group?